corehq/apps/accounting/emails.py

```diff
@@ -6,8 +6,14 @@
 from corehq.util.global_request import get_request
 
 
-def send_subscription_change_alert(domain, new_subscription, old_subscription, internal_change,
-                                   change_type='Change'):
+class SubjectTemplate:
+    CHANGE = "{env}Subscription Change Alert: {domain} from {old_plan} to {new_plan}"
+    RENEW = "{env}Subscription Renewal Alert: {domain} from {old_plan} to {new_plan}"
+    SELF_START = "{env}New Self-Start Subscription Alert: {domain} to {new_plan}"
+
+
+def send_subscription_change_alert(domain, new_subscription, old_subscription, internal_change=False,
+                                   subject=SubjectTemplate.CHANGE):
```

Comment:
Maybe `template` instead of `subject`? `subject` sounds like it'll be a string.

Reply:
Good call. https://github.com/dimagi/commcare-hq/pull/35281/commits/e617622feb8fc9225583674e0bcd77efa496e8b1